test/Microsoft.Azure.Devices.Edge.Test.Common/DaemonConfiguration.cs

```diff
@@ -4,7 +4,6 @@ namespace Microsoft.Azure.Devices.Edge.Test.Common
 {
     using System;
     using System.IO;
-    using Serilog;
 
     public class DaemonConfiguration
```

Comment:
So, this starts with a config files and edits it?

Reply:
Yes, it gets created after edge has been successfully installed so there's already a config.yaml file there (and in the Windows case that file has already been updated with all the info it needs, except proxy).